modules/nw-multus-bond-cni-object.adoc

```diff
@@ -87,4 +96,41 @@ The following example configures a secondary network named `bond-net1`:
         "gateway": "10.56.217.1"
     }
 }
-----
\ No newline at end of file
+----
+
+The following example configures a secondary network named `bond-tlb-net` with the `xmitHashPolicy` feature enabled:
+
+[source,json]
+----
+{
+ "type": "bond",
+ "cniVersion": "0.3.1",
+ "name": "bond-tlb-net",
+ "mode": "tlb",         
+ "xmitHashPolicy": "layer2+3", <1>
+ "failOverMac": 0,         
+ "linksInContainer": true,
+ "miimon": "100",
+ "mtu": 1500,
+ "links": [
+       {"name": "net1"},
+       {"name": "net2"}
+   ],
+  "ipam": {
+        "type": "host-local",
+        "subnet": "10.57.218.0/24", 
+        "routes": [{
+        "dst": "0.0.0.0/0"
+        }],
+        "gateway": "10.57.218.1"
+    }
+}
+----
+
+<1> This parameter dictates how outgoing network traffic is distributed (load-balanced) across the active member interfaces `net1` and `net2` within the bond.
+
+* `layer2+3` means the hashing algorithm uses a combination of:
+
+** `layer 2` information meaning source and destination MAC addresses
+
+** `layer 3` information meaning source and destination IP addresses
```

Comment:
Could you use a definition list for these three items and please introduce the list? Currently, this is just after the callout and feels like it's "in the air":
```suggestion
`layer2+3`:: The hashing algorithm uses a combination of `layer 2` and `layer 3` information.

`layer 2`:: Source and destination MAC addresses.

`layer 3`:: Source and destination IP addresses.
```

Reply:
Layer 3 not supported in isolation so I was trying to explain what layer2+3 means I have combined into one sentence